include/mbedtls/net_sockets.h

```diff
@@ -143,6 +144,8 @@ int mbedtls_net_bind( mbedtls_net_context *ctx, const char *bind_ip, const char
  *                  can be NULL if client_ip is null
  *
  * \return          0 if successful, or
+ *                  MBEDTLS_ERR_NET_SOCKET_FAILED,
```

Comment:
Note about the documentation update commit: you can see what error codes each function returns with
```
grep -E 'MBEDTLS_ERR_NET_[A-Z_]+|^int |(net_prepare\|net_would_block)\(' library/net_sockets.c
```